src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -1718,23 +1723,35 @@ public void optionsRequestShouldRespondWithOriginalPolicyHeaders() {
     }
 
     @Test
-    public void biddersParamsShouldReturnBidderSchemas() throws IOException, JSONException {
+    public void biddersParamsShouldReturnBidderSchemas() throws JSONException {
+        // given
+        final Map<String, JsonNode> bidderNameToSchema = bidderDeps.stream()
```

Comment:
Let's not rely on `BidderDeps` but work with the contents of `static/bidder-params/` directory only instead.